exonum-java-binding/core/app_assembly_macos.xml

```diff
@@ -0,0 +1,45 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+    <id>exonum-java-binding-app</id>
+    <formats>
+        <format>zip</format>
+    </formats>
+    <dependencySets>
+        <dependencySet>
+            <outputDirectory>lib/java</outputDirectory>
+        </dependencySet>
+    </dependencySets>
+    <fileSets>
+        <fileSet>
+            <directory>rust/target/prepackage</directory>
+            <outputDirectory>lib/native</outputDirectory>
+            <includes>
+                <include>%regex[libstd.*]</include>
+            </includes>
+        </fileSet>
+    </fileSets>
+    <files>
+        <file>
+            <source>rust/target/debug/ejb-app</source>
+        </file>
+        <file>
+            <source>rust/ejb-app/log4j.xml</source>
+        </file>
+        <file>
+            <source>rust/ejb-app/TUTORIAL.md</source>
+        </file>
+        <file>
+            <source>rust/target/prepackage/LICENSES-THIRD-PARTY.TXT</source>
+        </file>
+        <file>
+            <source>rust/target/prepackage/LICENSES-THIRD-PARTY-NATIVE</source>
+        </file>
+        <file>
+            <source>rust/target/prepackage/LICENSE</source>
+        </file>
+        <file>
+            <source>rust/target/debug/libjava_bindings.dylib</source>
```

Comment:
The only difference from `app_assembly_linux` is this line 🤔 I suppose I can use env var here instead